Cargo.toml

```diff
@@ -10,7 +10,7 @@ members = [
 	"td-layout",
 	"td-logger",
 	"td-paging",
-	"td-uefi-pi",
```

Comment:
It would be better to keep it as `td-uefi-pi`, so we have a common prefix for all crates

Reply:
Done